### PR TITLE
Automatic formatting and style checking

### PR DESCRIPTION
**Pull request by gabriel-v**
Just to see how it looks like.

Apparently, the default is no trailing commas (or no commas at all?).

### CODE REVIEW

**Comment by marco-c** [comment]
Overall the style looks nice to me. @calixteman do you have any other preference?

File: .rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+unstable_features = true
+indent_style = "Block"
+use_field_init_shorthand = true
+reorder_imports = false
```
Comment (marco-c) [inline .rustfmt.toml]: I would keep the reordering, sometimes it decreases the size of the patches.

Comment (calixteman) [reply .rustfmt.toml]: Looks good to me too: +1

File: .rustfmt.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+unstable_features = true
+indent_style = "Block"
+use_field_init_shorthand = true
```
[the remark this answers is not in the document]
Comment (gabriel-v) [reply .rustfmt.toml]: https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#use_field_init_shorthand

Default: false
Stable: Yes

[the remark this answers is not in the document]
Comment (marco-c) [reply .rustfmt.toml]: OK, I'd prefer `false` here, so if that's the default let's leave it as `false`.

File: .rustfmt.toml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+unstable_features = true
+indent_style = "Block"
```
Comment (marco-c) [inline .rustfmt.toml]: This is the default, right? So we can avoid setting it here?

Comment (gabriel-v) [reply .rustfmt.toml]: That's correct. The config file is now empty, so I'm removing it.

**Pull request merged**